pkgs/applications/audio/mixxx/default.nix

```
@@ -116,6 +116,20 @@ mkDerivation rec {
     "--set LOCALE_ARCHIVE ${glibcLocales}/lib/locale/locale-archive"
   ];
 
+  cmakeFlags = [
```

Let's add explanation

```suggestion
  # mixxx installs udev rules to DATADIR instead of SYSCONFDIR
  # let's disable this and install udev rules manually via postInstall
  # see https://github.com/mixxxdj/mixxx/blob/2.3.0/CMakeLists.txt#L1381-L1392
  cmakeFlags = [
```